### ISSUE TITLE
[request] Use AGENIX_RULES instead of RULES (and maybe rename secrets.nix too)

### ISSUE THREAD

**Issue by giorgiga** [ryantm/agenix#204]
The names `RULES` and `secrets.nix` are very generic and unrelated to agenix, to the point that any conscientious programmer can't avoid adding comments like "_this variable holds the path to the agenix rules file_" or "_this is the agenix rules file_" (ok, maybe only OCD programmers are affected).

Could you switch to something like `AGENIX_RULES` and `agenix-rules.nix` in a future version? (maintaining backwards-compatibility, of course).

**Comment by giorgiga** [comment]
BTW: I don't have much experience with either go or nix, but this seems easy enough that I can do it if you are willing to accept a PR (and would rather not do it yourself, of course)

**Comment by giorgiga** [comment]
BTW (since I'm here I'm not opening a separate issue), a second environment variable for holding what ssh key to use (as per the `-i` option) would be great... since my `id_ed25519` is passphrase-protected I generated a key for agenix, but now I have to specify `-i ~/.ssh/agenix_ed25519` every time

**Comment by ryantm** [comment]
Yeah, I think I'd accept such a PR if it had a suitable migration plan and documentation.

**Comment by giorgiga** [comment]
That's great - I should be able to work on it this weekend.
